docs/elements/hx-disclosure/index.html

```diff
@@ -7,14 +7,17 @@
   elements/hx-reveal: <hx-reveal>
 ---
 {% extends 'element.njk' %}
+{% import '_utils.njk' as utils %}
```

Comment:
do we need this `import`?

Reply:
It doesn't look like it. Please remove.

Reply:
@CITguy brought this over from your p/r originally but i dont think we need this